src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -5,21 +5,43 @@
 import tempfile
 
 
-@pytest.fixture(scope='function')
-def session():
-    with nidcpower.Session('4162', '', False, 'Simulate=1, DriverSetup=Model:4162; BoardType:PXIe') as simulated_session:
+@pytest.fixture(scope='function', params=[False, True])
+def session(request):
```

Comment:
Consider calling the old (not independent channels) "legacy sessions" and the new (independent channels) just session.

Reply:
This fixture has two arguments. The first time it gets called its a "legacy session" and the second time it gets called it is an independent channels session.  I figured that would allow system tests that need to run on both session types to use the `session` fixture while tests that need to explicitly use independent channels can call the `independent_channels_session` fixture.

Reply:
One concern I have with these different session types is the confusion it causes for future contributors to figure out which session or sessions to use. Currently, there are 5 - session, multi_instrument_session, synchronized_channels_session, single_channel_session, and multiple_channel_session.

Can we just have one session (parameterized for `independent_channels` parameter) and other session kinds localized to within specific tests that are testing the specific combination?

Reply:
After the independent channels support is added, I think the need for adding tests that behave differently between legacy/new sessions will be minimal if not zero. @marcoskirsch would have a better idea.

Reply:
Really we are only testing the Python bindings, and not the behavior of the driver runtime. We care that the Python calls hit the driver in the right way and anything beyond that is out of scope.

So we should exercise the codepaths of `nidcpower.Session.__init__() and for everything else only use "independent channels" Session.

Reply:
This is captured in #1614